src/components/FindNamedEntitiesForm.vue

```diff
@@ -105,35 +129,39 @@ export default {
         this.$store.commit('indexing/formPipeline', value)
       },
       get() {
-        return this.$store.state.indexing.form.pipeline
+        return this.$store.state.indexing.form.pipeline.toUpperCase()
       }
     },
     defaultProject: {
       set(value) {
         this.$store.commit('indexing/formDefaultProject', value)
       },
       get() {
-        return this.$store.state.indexing.form.defaultProject || this.$config.get('defaultProject')
+        return this.projectName || this.$store.state.indexing.form.defaultProject || this.$config.get('defaultProject')
       }
     },
     showProjectSelector() {
       return this.$core.projects.length > 1 || this.defaultProject !== this.$config.get('defaultProject')
     }
   },
   async mounted() {
-    this.$wait.start('load ner pipelines')
+    if (this.$core.findProject(this.projectName)) {
+      this.defaultProject = this.projectName
+    }
+    this.$wait.start('load ner pipelinesccccc')
     const pipelines = await this.$store.dispatch('indexing/getNerPipelines')
     this.$set(this, 'pipelines', this.handlePipelinesTranslation(values(pipelines).map(lowerCase)))
     this.$wait.end('load ner pipelines')
   },
   methods: {
     async submitFindNamedEntities() {
-      this.disabled = true
       try {
         await this.$store.dispatch('indexing/submitFindNamedEntities')
+        this.$emit('submit', { error: false })
+      } catch (e) {
```

Comment:
If you name the error parameter `error` you can use it directly in the next instruction: `this.$emit('submit', { error })`